test/smoke/helpers/TestDataHelper.js

```diff
@@ -0,0 +1,29 @@
+const appeals = require('../props/properties').testAppeals;
+moment = require('moment');
+
+class testDataHelper extends codecept_helper {
+
+getTestAppeal(desc)
+  {
+  var result = [];
```

Comment:
We shouldn't be using `var` here as we're using ES6. Consider using `let` or `const`.